scripts/init.d-xapissl

```diff
@@ -40,10 +39,6 @@ fi
 [ $(which stunnel4) ] && STUNNEL=$(which stunnel4)
 [ -z ${STUNNEL} ] && STUNNEL=$(which stunnel)
```

Comment:
I don't think we've had an `stunnel4` binary for a long time; `STUNNEL=$(command -v stunnel)` could be used instead